node/subsystem-util/src/lib.rs

```diff
@@ -394,13 +394,10 @@ impl Validator {
 
 /// ToJob is expected to be an enum declaring the set of messages of interest to a particular job.
 ///
-/// Normally, this will be some subset of `Allmessages`, and a `Stop` variant.
+/// Normally, this will be some subset of `Allmessages`.
 pub trait ToJobTrait: TryFrom<AllMessages> {
```

Comment:
Given the changes already in this PR, it's not clear that we actually need `ToJobTrait` anymore. It could simplify subsystem definitions if their inbound channel accepted `AllMessages`: their matches on inbound messages would need to explicitly discard irrelevant ones instead of implicitly in the `TryFrom<AllMessages> for ToJob` definitions, but the total amount of code would still be reduced.